package.json

```diff
@@ -108,10 +115,10 @@
         "uuid": "^8.3.2",
         "web-vitals": "^1.1.1"
     },
-    "pre-commit": [
-        "lint",
-        "prettier"
-    ],
+    "lint-staged": {
+        "*.{js,ts}": [ "npm run lint"],
+        "*.{js,html,md,yml}": [ "npm run prettier:fix"]
```

Comment:
1. Our preference is that prettier check during pre-commit, rather than format during pre-commit. 
2. Please include typescript and json in this list.

Reply:
For the first one, I can update, but why not format on pre-commit?
Wouldn't that make it easier for people to make updates?

Also for number 2 -- will add, was a copy/paste from docs.